src/front/app/components/ChallengeTopics/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { getChallengeTopics, toggleChallengeTopic } from './../../actions/challengeTopics';
+import ChallengeTopic from './ChallengeTopic';
+
+class ChallengeTopicList extends Component {
+    constructor(props) {
+        super(props);
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    componentDidMount() {
+        const { dispatch } = this.props;
+        dispatch(getChallengeTopics());
+    }
+
+    handleClick(name) {
```

Comment:
to minimize impact on component change 
cases
- **name** changed to **shortName**
- some-who will require not just name in created reducer